airflow/www/templates/airflow/providers.html

```diff
@@ -0,0 +1,49 @@
+{#
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+#}
+
+{% extends base_template %}
+
+{% block title %}
+  {{ title }}
+{% endblock %}
+
+{% block content %}
+  <h2>{{ title }}</h2>
+  {% if providers|length == 0 %}
+    <p>No providers loaded. Learn more in the
+      <a href="{{ doc_url }}" target="_blank">provider packages documentation</a>.
+    </p>
+  {% endif %}
+
+  <table class="table table-striped table-bordered">
+    <tr>
+      <th>Package Name</th>
+      <th>Version</th>
+      <th>Description</th>
+    </tr>
+
+    {% for provider in providers %}
+      <tr>
+          <td>{{ provider["package_name"] }}</td>
+          <td>{{ provider["version"] }}</td>
+          <td>{{ provider["description"] | safe}}</td>
```

Comment:
```suggestion
          <td>{{ provider["description"]|safe}}</td>
```
Doesn't _really_ matter, but this makes it consistent with line 28.